### PR TITLE
1135210 An empty search on desktop should not say "" returned X results

### PR DESCRIPTION
**Pull request by dipankarsonwane**

### CODE REVIEW

File: src/templates/search.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
 {% include '_macros/more_button.html' %}
 
 {% macro results_subheader(query, count) %}
+  {% if query=="" %}
+    {% set query = "Everything" %}
```
Comment (ngokevin) [inline src/templates/search.html]: You can send in ```query || gettext('Everything')``` through the search.js view.

File: src/templates/search.html [hunk 2]
```diff
@@ -11,7 +14,7 @@ <h1 class="search-results-header search-results-header-mobile">
       <h1 class="search-results-header search-results-header-desktop">
         {{ _plural('<b>"{search_query}"</b> returned {n} result',
                    '<b>"{search_query}"</b> returned {n} results',
-                   search_query=query|escape,
+                   search_query=query|safe,
```
Comment (mstriemer) [inline src/templates/search.html]: I don't think this is a safe string. You should be able to leave this as it was.

Comment (gurumukhi) [inline src/templates/search.html]: @mstriemer with escape the code changes made at line 5 & 6 were not working. Can you please advice anything regarding this if we keep it escape itself

File: src/templates/search.html (hunk 1 above)
Comment (gurumukhi) [inline src/templates/search.html]: @dipankarsonwane can you please make this change ?

Comment (mstriemer) [inline src/templates/search.html]: This should actually be `query || _(Everything')`

File: src/templates/search.html [hunk 3]
```diff
@@ -2,6 +2,9 @@
 {% include '_macros/more_button.html' %}
 
 {% macro results_subheader(query, count) %}
+  {% if query=="" %}
```
Comment (mstriemer) [inline src/templates/search.html]: There should be spaces around the `==`:

```jinja
{% if query == "" %}

File: src/templates/search.html (hunk 2 above)
Comment (mstriemer) [inline src/templates/search.html]: Just pulled this code down and it works fine with `escape` on my machine.

File: src/templates/search.html (hunk 1 above)
Comment (ngokevin) [inline src/templates/search.html]: I'd rather have it in the JS than the template though.

Comment (mstriemer) [inline src/templates/search.html]: Ah, totally. I missed that you suggested moving it. Ignore me.

**Comment by ngokevin** [comment]
Going to close since the solution is off, but thanks and feel free to address the comments in another PR.

**Comment by gurumukhi** [comment]
@dipankarsonwane did you submit another PR with comments given above ?

**Comment by dipankarsonwane** [comment]
No

**Comment by gurumukhi** [comment]
Let me know if you are facing any issues in submitting another PR. in case you are not able to work on this and dont want to continue, you can mention this on bug, someone else will pick this up then

**Pull request closed without merging**